### PR TITLE
Darrich/demoadd

### PR DESCRIPTION
**Pull request by ShadowPic**
## Purpose

Checking in the example code with redacted sections

## Does this introduce a breaking change?

```
[ ] Yes
[x ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code
* Follow the instructions in the readme.md

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Comment by ShadowPic** [comment]
can anyone review this?

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,164 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
```
Comment (trask) [inline azure-pipelines.yml:4]: is this pipeline being used?

File: source/azure-mongodb-error-handling-demo/.gitignore [hunk 2]
```diff
@@ -0,0 +1,33 @@
+HELP.md
+target/
+!.mvn/wrapper/maven-wrapper.jar
+!**/src/main/**/target/
+!**/src/test/**/target/
```
Comment (trask) [inline source/azure-mongodb-error-handling-demo/.gitignore:5]: probably only need the root `.gitignore`?

File: source/jfr2jsonfunction/JfrToJson.sln [hunk 3]
```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.30907.101
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "JfrToJson", "JfrToJson\JfrToJson.csproj", "{9203B1A8-AC25-4C76-93C3-B8158094F02C}"
```
Comment (trask) [inline source/jfr2jsonfunction/JfrToJson.sln:6]: I'd suggest removing IDE file if it's not necessary

File: source/jfr2jsonfunction/JfrToJson/.gitignore [hunk 4]
```diff
@@ -0,0 +1,264 @@
+## Ignore Visual Studio temporary files, build results, and
+## files generated by popular Visual Studio add-ons.
+
+# Azure Functions localsettings file
+local.settings.json
```
Comment (trask) [inline source/jfr2jsonfunction/JfrToJson/.gitignore:5]: probably only need root `.gitignore`

File: workspace.code-workspace [hunk 5]
```diff
@@ -0,0 +1,11 @@
+{
+	"folders": [
+		{
+			"path": "."
+		}
+	],
+	"settings": {
+		"debug.internalConsoleOptions": "neverOpen",
+		"powershell.codeFormatting.addWhitespaceAroundPipe": true
+	}
```
Comment (trask) [inline workspace.code-workspace:10]: same suggestion for IDE file(s)

File: source/jfr2jsonfunction/JfrToJson/Function1.cs [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Azure.Storage;
+//using Newtonsoft.Json;
+using System.Diagnostics;
+using System.Text.Json;
+
+namespace JfrToJson
```
Comment (trask) [inline source/jfr2jsonfunction/JfrToJson/Function1.cs:14]: what is this .NET code? 😂

**Review by trask — COMMENTED** [review]
thanks @ShadowPic!

some comments below.

also I'd suggest removing the `.DS_Store` file

and `source/azure-mongodb-error-handling-demo/src/main/resources/application.properties` is empty, is it needed?

File: azure-pipelines.yml (hunk 1 above)
Comment (ShadowPic) [reply azure-pipelines.yml:4]: It's demonstration pipeline that does with it.

File: source/azure-mongodb-error-handling-demo/.gitignore (hunk 2 above)
Comment (ShadowPic) [reply source/azure-mongodb-error-handling-demo/.gitignore:5]: probably but doesn't hurt.  :)

**Comment by trask** [comment]
thx @ShadowPic!

**Pull request merged**